doc/classes/Image.xml

```diff
@@ -710,7 +710,7 @@
 		</constant>
 		<constant name="INTERPOLATE_TRILINEAR" value="3" enum="Interpolation">
 			Performs bilinear separately on the two most-suited mipmap levels, then linearly interpolates between them.
-			It's slower than [constant INTERPOLATE_BILINEAR], but produces higher-quality results with much less aliasing artifacts.
+			It's slower than [constant INTERPOLATE_BILINEAR], but produces higher-quality results with much fewer aliasing artifacts.
```

Comment:
```suggestion
			It's slower than [constant INTERPOLATE_BILINEAR], but produces higher-quality results with far fewer aliasing artifacts.
```
Honestly the "much less" sounds almost as good to me though.

Reply:
"much less" is grammatically incorrect because it's using the wrong quantifier in the given context. This is because "artifacts" is a countable plural noun.